src/MissionEditor/SurveyItemEditor.qml

```diff
@@ -115,6 +152,79 @@ Rectangle {
             onPolygonAdjustVertex: missionItem.adjustPolygonCoordinate(vertexIndex, vertexCoordinate)
         }
 
+        QGCLabel { text: qsTr("Camera:") }
+
+        Rectangle {
+            anchors.left:   parent.left
+            anchors.right:  parent.right
+            height:         1
+            color:          qgcPal.text
+        }
+
+        Row {
+            spacing: ScreenTools.defaultFontPixelWidth
+
+
+            QGCRadioButton {
+                id:             cameraOrientationLandscape
+                text:           "Landscape"
+                checked:        true
+                exclusiveGroup: cameraOrientationGroup
+            }
+
+            QGCRadioButton {
+                id:             cameraOrientationPortrait
+                text:           "Portrait"
+                exclusiveGroup: cameraOrientationGroup
+            }
+        }
+
+        Grid {
+            columns: 2
+            spacing: ScreenTools.defaultFontPixelWidth
+            verticalItemAlignment: Grid.AlignVCenter
+
+            QGCLabel { text: qsTr("Focal length:") }
+            QGCTextField {
+                id:         focalLengthField
+                unitsLabel: "mm"
+                showUnits:  true
+                text:       _cameraInfoCanonSX260.focalLength.toString()
+
+                onEditingFinished: recalcFromCameraValues()
+            }
+
+            QGCLabel { text: qsTr("Sensor Width:") }
+            QGCTextField {
+                id:         sensorWidthField
+                unitsLabel: "mm"
+                showUnits:  true
+                text:       _cameraInfoCanonSX260.sensorWidth.toString()
+
+                onEditingFinished: recalcFromCameraValues()
+            }
+
+            QGCLabel { text: qsTr("Sensor height:") }
+            QGCTextField {
+                id:         sensorHeightField
+                unitsLabel: "mm"
+                showUnits:  true
+                text:       _cameraInfoCanonSX260.sensorHeight.toString()
+
+                onEditingFinished: recalcFromCameraValues()
+            }
+
+            QGCLabel { text: qsTr("Image overlap:") }
```

Comment:
This is a cool feature! I would suggest to have separate front and side overlap.